datasource/src/main/scala/cromwell/pipeline/database/TestMongoEngine.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.pipeline.database
+
+import cromwell.pipeline.utils.MongoConfig
+import org.mongodb.scala.{ MongoClientSettings, ServerAddress }
+
+import scala.collection.JavaConverters._
+
+class TestMongoEngine(mongoConfig: MongoConfig) extends MongoEngine(mongoConfig) {
```

Comment:
Why is `TestMongoEngine` not in `test` directory?

Reply:
Fixed